content/rc/subscriptions/create-fixed-subscription.md

```diff
@@ -10,31 +10,49 @@ aliases: /rv/administration/setup_and_editing/create-subscription/
          /rc/administration/setup-and-editing/create-subscription/
          /rc/administration/setup/create-subscription/
 ---
-Fixed Size subscription plans support low throughput workflows.  Several tiers are available, each designed for different memory sizes and integration requirements.
+Fixed Size subscription plans support low throughput workflows.  Several plans are available, each designed for different memory sizes.
 
-When creating your subscription, you'll need to know which tier to choose.
+When creating your subscription, you'll need to choose the plan that suits your needs.
 
 If you're new to Redis Cloud, the [quick start]({{<relref "/rc/rc-quickstart.md">}}) helps you create an account with a free subscription and an initial database.  You also learn how to connect to your database.
 
+## Fixed plans
 
-## Fixed plan subscription tiers
+Fixed plan pricing scales according to the memory size of the database defined in the subscription.  Additional limits also apply, as shown in the tables below (updated February 2024). 
 
-Fixed plan pricing scales according to the memory size of the database defined in the subscription.  Additional limits also apply, as shown here (updated January 2024):
+The 30 MB Fixed plan is free; it's designed for learning and building test projects. It gives you enough space to learn Redis concepts and develop application prototypes.
 
-| **Max DB size &nbsp;** | **Concurrent<br/>connections<br/>per database** | **CIDR<br/> allow rules** |
-|---|---|---|
-| (Free) 30 MB &nbsp;&nbsp;&nbsp;&nbsp; | 30 | 1 |
-| 250 MB &nbsp;&nbsp;&nbsp;&nbsp; | 256 | 4-8 |
-| 1 GB &nbsp;&nbsp;&nbsp;&nbsp; | 1024 | 4-8 |
-| 2.5 GB &nbsp;&nbsp;&nbsp;&nbsp; | Unlimited | 4-8 |
-| 5 GB &nbsp;&nbsp;&nbsp;&nbsp; | Unlimited | 4-16 |
-| 12 GB &nbsp;&nbsp;&nbsp;&nbsp; | Unlimited | 4-32 |
+Our paid plans start at 250 MB and offer extra features, like multi-zone replication and backups (both daily and instant). They are great for bigger projects or production environments that require increased storage, reliability, and other features to support your operational needs.
 
-The 30 MB Fixed plan is free; it's designed for training and prototype purposes.
+If you need additional resources, you can [upgrade your subscription]({{<relref "/rc/subscriptions/view-fixed-subscription#upgrade-subscription-plan">}}) at any time.
 
-All paid (250 MB and above) Fixed plans support replication and backups (daily and instant).
+There are some differences between plans for the different replication options. See the tables for [No replication and single-zone replication](#cache-standard) and [Multi-zone replication](#multi-az) for more details.
```

Comment:
@cmilesb Also here, I think we should use high availability rather than replication, especially since in the new packaging UI we use the former.